rust/src/ics23.rs

```diff
@@ -36,8 +36,8 @@ pub struct ExistenceProof {
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct NonExistenceProof {
     /// TODO: remove this as unnecessary??? we prove a range
-    #[prost(bytes, tag = "1")]
-    pub key: alloc::vec::Vec<u8>,
+    #[prost(bytes = "vec", tag = "1")]
+    pub key: ::prost::alloc::vec::Vec<u8>,
```

Comment:
This looks better. I need to test it locally

Reply:
> This looks better. I need to test it locally

enen,  I tested it and it's OK.